packages/chat-headless/package.json

```diff
@@ -40,7 +40,7 @@
   "dependencies": {
     "@reduxjs/toolkit": "^1.9.5",
     "@yext/analytics": "^0.6.3",
-    "@yext/chat-core": "^0.5.4"
+    "@yext/chat-core": "^0.6.0"
```

Comment:
might as well bump this to 0.6.1 for the new fix!

Reply:
oop, good call!